src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/SplitButton/SplitButtonTests.cs

```diff
@@ -82,6 +85,25 @@ public void VerifyIsCheckedProperty()
 				Verify.IsTrue(isChecked, "ToggleSplitButton is not checked");
 			});
 		}
+
+		[TestMethod]
+		[Description("Verifies that the TextBlock representing the Chevron glyph uses the correct font")]
+		public void VerifyFontFamilyForChevron()
+		{
+			SplitButton splitButton = null;
+			using (StyleHelper.UseFluentStyles())
```

Comment:
I suspect that this `using` should be inside the UI thread block, since it's modifying App.Resources. In general it's easier just to use the `[RunsOnUIThread]` attribute for the whole test :)

Reply:
Will fix.

Reply:
@davidjohnoliver Sorry missed this for a while. Pushed a commit using the attribute.